<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetectionSpeedDlg</class>
 <widget class="QDialog" name="DetectionSpeedDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QToolButton" name="toolButton">
      <property name="geometry">
       <rect>
        <x>285</x>
        <y>0</y>
        <width>24</width>
        <height>23</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../res.qrc">
        <normaloff>:/images/help.png</normaloff>:/images/help.png</iconset>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>21</y>
        <width>291</width>
        <height>219</height>
       </rect>
      </property>
      <property name="title">
       <string>监测速度</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>42</y>
         <width>34</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>右前:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>32</x>
         <y>42</y>
         <width>34</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>左前:</string>
       </property>
      </widget>
      <widget class="QLabel" name="LFLabel">
       <property name="geometry">
        <rect>
         <x>72</x>
         <y>46</y>
         <width>46</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#204a87;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="RFLabel">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>46</y>
         <width>46</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#204a87;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>32</x>
         <y>79</y>
         <width>34</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>左后:</string>
       </property>
      </widget>
      <widget class="QLabel" name="RBLabel">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>82</y>
         <width>46</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#204a87;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>79</y>
         <width>34</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>右后:</string>
       </property>
      </widget>
      <widget class="QLabel" name="LBLabel">
       <property name="geometry">
        <rect>
         <x>72</x>
         <y>82</y>
         <width>46</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#204a87;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>116</y>
         <width>36</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>米/秒</string>
       </property>
      </widget>
      <widget class="QLabel" name="speedLabel">
       <property name="geometry">
        <rect>
         <x>133</x>
         <y>120</y>
         <width>31</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#204a87;&quot;&gt;0.0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>93</x>
         <y>116</y>
         <width>34</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>速度:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="closeButton">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>178</y>
         <width>80</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
      <widget class="QPushButton" name="stopButton">
       <property name="geometry">
        <rect>
         <x>104</x>
         <y>178</y>
         <width>80</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
      <widget class="QPushButton" name="startButton">
       <property name="geometry">
        <rect>
         <x>18</x>
         <y>178</y>
         <width>80</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
